<?xml version="1.0" encoding="utf-8"?>
<report>
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="4a4fdf42d882bb2"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="automation"
		host="DESKTOP-IP0304R"
		rxversion="5.4.0.24580"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		language="en-US"
		screenresolution="1024x768"
		timestamp="3/20/2016 12:37:16 PM"
		timestampiso="2016-03-20T16:37:16"
		timeoutfactor="1"
		module="OpenWebSite">
		<detail>
			OpenWebSite
		</detail>
		<varbindings/>
		<item
			timeRelativeToTestSuiteStartTime="00:21.726"
			timeRelativeToTestModuleStartTime="00:21.726"
			timeWallClock="12:37:37"
			time="00:21.726"
			level="Info"
			category="User">
			<message>
				10 20
			</message>
			<metainfo
				codefile="c:\RanorexSession2_Cl\Session2_CL\Session2_CL\OpenWebSite.cs"
				codeline="53"
				loglvl="Info"/>
		</item>
		<item
			timeRelativeToTestSuiteStartTime="17:06.282"
			timeRelativeToTestModuleStartTime="17:06.282"
			timeWallClock="12:54:22"
			time="17:06.282"
			level="Error"
			category="Module">
			<message>
				Test run was aborted by the user.
			</message>
			<metainfo
				loglvl="Error"/>
		</item>
	</activity>
</report>
